documentation/modules/ROOT/pages/configuration/applying-transformations-selectively.adoc

```diff
@@ -0,0 +1,143 @@
+// Category: debezium-using
+// Type: assembly
+// ModuleId: applying-transformations-selectively-with-smt-predicates
+// Title: Applying transformations selectively with SMT predicates
+= Applying transformations selectively
+
+You can define a predicate for a single message transformation to specify a condition that event records must match before Kafka Connect applies the transformation.
+
+// Type: concept
+// ModuleID: about-smt-predicates
+// Title: About SMT predicates
+== SMT predicates
+
+{prodname} provides several single message transformations (SMTs) that you can use to modify event records before Kafka Connect saves the records to Kafka topics.
```

Comment:
SMTs can be applied on the source side (i.e. what you describe here) and on the sink side (i.e. when reading events from a Kafka topic, before passing them to the sink connector). Not sure how to describe this here concisely.

Reply:
@roldanbob, so "Kafka Connect saves the records to Kafka topic" still doesn't quite reflect the ability ot use SMTs with sink connectors.

Reply:
@gunnarmorling In discussing predicates here I want to keep the focus on how they're used with source connectors such as Debezium. But I did add a paragraph further on that describes a bit more about their use with sink connectors:

> The situation is similar for predicates that you define for a sink connector SMT.
> The connector reads messages from a Kafka topic and Kafka Connect evaluates the messages against the predicate condition.
> If a message matches the condition, Kafka Connect applies the transformation and then passes the messages to the sink consumer.

Does that address your concern?